plugins/russian/authortoday.ts

```diff
@@ -4,23 +4,22 @@ import { defaultCover } from '@libs/defaultCover';
 import { fetchApi, fetchFile } from '@libs/fetch';
 import { NovelStatus } from '@libs/novelStatus';
 import { load as parseHTML } from 'cheerio';
+import { storage } from '@libs/storage';
 import dayjs from 'dayjs';
 
-const apiUrl = 'https://api.author.today/';
-const token = 'Bearer guest';
-
 class AuthorToday implements Plugin.PluginBase {
   id = 'AT';
   name = 'Автор Тудей';
   icon = 'src/ru/authortoday/icon.png';
   site = 'https://author.today';
-  version = '1.0.0';
+  apiUrl = 'https://api.author.today/v1/';
+  version = '1.0.1';
```

Comment:
missed storage flag

Reply:
The `webStorageUtilized` flag is needed to access LocalStorage, SesesionStorage

Reply:
Oh, then I think we can add that lfag in app side, by checking plugin requries

Reply:
I've already added this.
https://github.com/LNReader/lnreader/blob/89a1b7265734d3512e1277d727629d9d2d923487/src/plugins/types/index.ts#L70
https://github.com/LNReader/lnreader/blob/89a1b7265734d3512e1277d727629d9d2d923487/src/screens/WebviewScreen/WebviewScreen.tsx#L26
https://github.com/LNReader/lnreader/blob/89a1b7265734d3512e1277d727629d9d2d923487/src/screens/WebviewScreen/WebviewScreen.tsx#L49-L60